### ISSUE TITLE
Phone icon opens contacts

### ISSUE THREAD

**Issue by ronaldkaptein** [sduduzog/slim-launcher#62]
When I click the phone icon, the contacts app is opened. I can dial a contact from here, but not dial in a number or see recent calls. I'd like to open the Phone app instead. The problem might be that on my phone (Samsung Galaxy S7) the contact app is set as default calling app, and I cannot choose the phone app. A solution might be to add a setting allowing to chose the app that is opened when the phone icon is clicked.

**Comment by ribab** [comment]
Yeah, my phone doesn't have that issue since I can dial straight from the app that button opens
